A test for plotting fails

## Step 2: Describe your environment

  * Operating system: Ubuntu 18.04 LTS
  * Python Version: 3,6,8 (`python -V`)
  * CCXT version: 1.18.1115 (`pip freeze | grep ccxt`)
  * Branch: Develop
  * Last Commit ID: 242ac4d8f484c5aa23e2266ab9ebe335153390e4 (`git log --format="%H" -n 1`)
  
## Step 3: Describe the problem:
Pytest fails at my site:

```
================================================================== FAILURES ==================================================================
________________________________________________________ test_start_plot_profit_error ________________________________________________________

mocker = <pytest_mock.MockFixture object at 0x7fd3ceb69160>

    def test_start_plot_profit_error(mocker):
        args = [
            "plot-profit",
            "--pairs", "ETH/BTC"
        ]
        with pytest.raises(OperationalException):
>           start_plot_profit(get_args(args))

tests/test_plotting.py:344: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
freqtrade/plot/plot_utils.py:36: in start_plot_profit
    plot_profit(config)
freqtrade/plot/plotting.py:376: in plot_profit
    plot_elements = init_plotscript(config)
freqtrade/plot/plotting.py:49: in init_plotscript
    exportfilename=config.get('exportfilename'),
freqtrade/data/btanalysis.py:124: in load_trades
    return load_backtest_data(Path(exportfilename))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

filename = PosixPath('user_data/backtest_results/backtest-result.json')

    def load_backtest_data(filename) -> pd.DataFrame:
        """
        Load backtest data file.
        :param filename: pathlib.Path object, or string pointing to the file.
        :return: a dataframe with the analysis results
        """
        if isinstance(filename, str):
            filename = Path(filename)
    
        if not filename.is_file():
>           raise ValueError(f"File {filename} does not exist.")
E           ValueError: File user_data/backtest_results/backtest-result.json does not exist.

freqtrade/data/btanalysis.py:33: ValueError
------------------------------------------------------------- Captured log call --------------------------------------------------------------
INFO     freqtrade.configuration.configuration:configuration.py:64 Using config: config.json ...
INFO     freqtrade.configuration.configuration:configuration.py:74 Validating configuration ...
INFO     freqtrade.loggers:loggers.py:50 Verbosity set to 0
INFO     freqtrade.configuration.configuration:configuration.py:137 Dry run is enabled
INFO     freqtrade.configuration.configuration:configuration.py:146 Using DB: "sqlite://"
INFO     freqtrade.configuration.configuration:configuration.py:204 Using max_open_trades: 3 ...
INFO     freqtrade.configuration.configuration:configuration.py:177 Using user-data directory: /home/user/freqtrade-wrk/github-hroff-1902/freqtrade/user_data ...
INFO     freqtrade.configuration.configuration:configuration.py:183 Using data directory: /home/user/freqtrade-wrk/github-hroff-1902/freqtrade/user_data/data/bittrex ...
INFO     freqtrade.configuration.configuration:configuration.py:333 Storing backtest results to user_data/backtest_results/backtest-result.json ...
INFO     freqtrade.configuration.configuration:configuration.py:333 Using pairs ['ETH/BTC']
INFO     freqtrade.configuration.configuration:configuration.py:333 Using trades from: file
INFO     freqtrade.configuration.check_exchange:check_exchange.py:27 Checking exchange...
INFO     freqtrade.configuration.check_exchange:check_exchange.py:43 Exchange "bittrex" is officially supported by the Freqtrade development team.
INFO     freqtrade.data.history:history.py:162 Using local backtesting data ...
INFO     freqtrade.data.converter:converter.py:91 Missing data fillup for ETH/BTC: before: 5760 - after: 6065
```

--- a/tests/test_plotting.py
+++ b/tests/test_plotting.py
@@ -336,12 +336,17 @@ def test_start_plot_profit(mocker):
 
 
 def test_start_plot_profit_error(mocker):
+
     args = [
         "plot-profit",
         "--pairs", "ETH/BTC"
     ]
+    argsp = get_args(args)
+    # Make sure we use no config. Details: #2241
+    # not resetting config causes random failures if config.json exists
+    argsp.config = []
     with pytest.raises(OperationalException):
-        start_plot_profit(get_args(args))
+        start_plot_profit(argsp)
 
 
 def test_plot_profit(default_conf, mocker, testdatadir, caplog):

